packages/material-ui/src/RadioWidget/RadioWidget.tsx

```diff
@@ -43,11 +45,16 @@ export default function RadioWidget<T = any, S extends StrictRJSFSchema = RJSFSc
   const row = options ? options.inline : false;
   const selectedIndex = enumOptionsIndexForValue<S>(value, enumOptions);
 
+  const { schemaUtils } = registry;
+  const displayLabel = schemaUtils.getDisplayLabel(schema, uiSchema);
```

Comment:
We added a way to turn off labels globally, so updating this
```suggestion
  const { schemaUtils, globalUiOptions } = registry;
  const displayLabel = schemaUtils.getDisplayLabel(schema, uiSchema, globalUiOptions);
```